lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -58,9 +58,11 @@ const rule = (primary, secondaryOptions) => {
 		);
 
 		eachDeclarationBlock(root, (eachDecl) => {
-			/** @type {string[]} */
+			/** @type {import('postcss').Declaration[]} */
 			const decls = [];
 			/** @type {string[]} */
+			const declsProps = [];
+			/** @type {string[]} */
 			const values = [];
 
 			eachDecl((decl) => {
```

Comment:
[suggest] It seems possible to replace a new variable `decls` with the third argument of `eachDecl()` like this:

```suggestion
			eachDecl((decl, _index, decls) => {
```

Reply:
Ah, the new variable `decls` can be a smaller array than the third argument `decls`.
Sorry, keep the current code as-is.